demisto_sdk/main.py

```diff
@@ -172,15 +167,17 @@ def validate(config, **kwargs):
     sys.path.append(config.configuration.env_dir)
 
     validator = FilesValidator(configuration=config.configuration,
-                               is_backward_check=str2bool(kwargs['backward-comp']),
-                               is_circle=str2bool(kwargs['circle']), prev_ver=kwargs['prev_ver'],
+                               is_backward_check=kwargs['backward_comp'],
+                               is_circle=kwargs['post_commit'], prev_ver=kwargs['prev_ver'],
                                validate_conf_json=kwargs['conf_json'], use_git=kwargs['use_git'])
     return validator.run()
 
 
 # ====================== create ====================== #
 @main.command(name="create",
```

Comment:
we are moving this after all? if not we should discuss this first , and if this command stays they definitely the name should be changed. it is not enough to change the description i

Reply:
Lets talk about that tomorrow, but i think that we can probably move that back into the content repo.
@yaakovi what do you think about this?

Reply:
it is not something that one will actually use (beside content developers).
this feature can stay in the sdk if we want to support custom content zip files that one can generate and upload.